### DISCUSSION TITLE
Support for Magisk 24.0 or higher in emulator

### DISCUSSION THREAD

**Discussion by MohamedHwsh** [Genymobile/feedback#55] [Genymotion Desktop] [answered]
As per the article: [How to install magisk on genymotion](https://support.genymotion.com/hc/en-us/articles/8957952431389-How-to-install-Magisk-on-Genymotion-) by Genymotion team, the magisk framework is outdated ver 23.0,
If possible, provide instructions on how to build magisk framework for genymotion, or provide an updated binary of it.

The reason I ask this is because I need ver 24.0 or higher to be able to use a feature called “Zygest”. Which I intend to use with LSposed.

-
I have tried to find a boot image for genymotion to patch and use for magisk, but to no avail, also I tried to extract the binary from the Magisk apk, but I have seen genymotion get into bootloops.

Thanks.

**Comment by jmaneyrol69** [comment, marked as answer]
duplicate of #70
